.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "staging/docs-6", "develop", "feature/TinyMCE_5.10", "feature/TinyMCE_5.10.3", "main", "release/TinyMCE_5.10", "release/docs-6" ]
```

Comment:
I think we should probably just use `feature/**` and `release/**` here. Although I still haven't double-checked which branches we even need to put here.

Reply:
What about `staging/**`?